cmd/svcat/instance/provision_cmd.go

```diff
@@ -79,6 +84,11 @@ func NewProvisionCmd(cxt *command.Context) *cobra.Command {
 		"Additional parameter, whose value is stored in a secret, to use when provisioning the service, format: SECRET[KEY]")
 	cmd.Flags().StringVar(&provisionCmd.jsonParams, "params-json", "",
 		"Additional parameters to use when provisioning the service, provided as a JSON object. Cannot be combined with --param")
+	cmd.Flags().BoolVar(&provisionCmd.wait, "wait", false,
```

Comment:
Why not bump these out to the top level flags? are there commands where this does not apply? Or a common function to call to add to cmd.Flags() to mix-in the wait flags.

Reply:
I don't think it should be a top level flag because --wait for `svcat get plans` for example won't be implemented.

I will do the same thing I did for namespace's helper functions, so that there is a common interface for commands that support waiting.

Reply:
if the broker is just a proxy, even get plans might take awhile... just saying OSB is flexible.

Reply:
When I run `svcat get plans`, I am hitting the service catalog api, not the broker, so if the broker isn't finished listing (because it's slow or something), then I get nothing back. If I wanted to wait on something I would use `svcat get broker slowassbroker --wait` until it flips from an empty status/errfetchingcatalog to ready.

The idea behind wait is to wait for a mutating operation on a resource to complete (successfully or otherwise). I'm not sure that it applies to idempotent list operations. If we wanted something like that, I'd want to hash out what the UX would be and probably use a differently named flag for that. You know?